docs/include/conf/current.rst

```diff
@@ -1,7 +1,5 @@
 2025 Conferences
```

Comment:
We normally keep all the conferences for the year on the front page, but we could add a banner or highlight Berlin first in the list?

There's some code in the conf.py file from the banner we put in Portland that I'm guessing should be easy to reuse.

Reply:
Sounds good. I kept it on the front page, but broke the list in two - making the order not chronological looked odd too. Banner added.